eventkit_cloud/api/filters.py

```diff
@@ -83,14 +83,26 @@ class Meta:
 
     @staticmethod
     def group_filter( queryset, fieldname, value):
-
-        targetusers = []
-        perms  = GroupPermission.objects.filter(group__in=value.split(","))
-        for perm in perms:
-            user = perm.user
-            if not user.id in targetusers: targetusers.append(user.id)
-
-        return queryset.filter(id__in=targetusers)
+        target_users = []
+        groups = value.split(",")
+
+        # Find any users who are not in any groups
```

Comment:
The comments here should explain and example query i.e. `users?groups=none,1` and why we need this functionality to prevent regressions.